packages/components/label/src/required-indicator.tsx

```diff
@@ -5,6 +5,7 @@ const RequiredIndicator = () => (
   <em
     css={css`
       color: ${designTokens.colorWarning};
+      font-style: ${designTokens.fontStyleForRequiredIndicator};
```

Comment:
I would expect us just to use `normal` here since we don't want to use _italic_ for this indicator in any of the themes.